app/pages/events/race_registered_list.php

```diff
@@ -0,0 +1,58 @@
+<?php
+restrict_access(Access::$ADD_EVENTS);
+
+$event_id = get_route_param("event_id");
+$race_id = get_route_param("race_id", false);
+$event = em()->find(Event::class, $event_id);
+$race = em()->find(Race::class, $race_id);
```

Comment:
Good idea ! Do you think we can also apply that to other race pages ?

Reply:
Maybe 🤔 I think projecting is a good practice in general. Maybe we should consider disabling lazy loading altogether.

(I know that for example my company has lazy-loading disabled to prevent flooding the db server with extra queries. It's not the same scale, but doing it now would prevent massive refactoring later)

Reply:
Maybe we don't extend this further than the scope of this PR, but in general try to reduce the amount of querying you are doing to the DB.
With the query I suggested, it goes down to 1.
Without it, you are making 3 queries with an extra query for **every single entry**